input-explainer.md

```diff
@@ -235,19 +233,19 @@ function onSelectStart(event) {
   
   let inputPose = event.frame.getInputPose(event.inputSource, xrReferenceSpace);
   // Ignore the event if this input source is not capable of tracking.
-  if (!inputPose || !inputPose.gripMatrix)
+  if (!inputPose || !inputPose.grip)
     return;
 
   // Use the input source target ray to find a draggable object in the scene
   let ray = inputPose.targetRay;
   let hitResult = scene.hitTest(ray.origin, ray.direction);
   if (hitResult && hitResult.draggable) {
-    // Use the gripMatrix translation to drag the intersected object, rather than the target ray.
+    // Use the grip translation to drag the intersected object, rather than the target ray.
     activeDragInteraction = {
       target: hitResult,
       targetStartPosition: hitResult.position,
       inputSource: event.inputSource,
-      inputSourceStartPosition: getTranslation(inputPose.gripMatrix);
+      inputSourceStartPosition: inputPose.grip.translation;
```

Comment:
This line is a smell that `XRRigidTransform.position` naming would be more natural than `translation`, as this is being immediately assigned to an app attribute named `inputSourceStartPosition` - the app thinks of the return value here as a position.

Reply:
Feels like this this is a simplified special case, to me.  The fact that this particular transform is used as something that feels like an absolute value (i.e., "position") versus something that feels like a relative value (i.e., "translation") shouldn't dictate the term.  

To me, a RigidTransform feels relative (i.e., translation / rotation) because they can be composed, even if it's not something that gets done in these special cases.